features/call/impl/src/main/kotlin/io/element/android/features/call/impl/ui/CallScreenPresenter.kt

```diff
@@ -68,6 +70,7 @@ class CallScreenPresenter @AssistedInject constructor(
     private val screenTracker: ScreenTracker,
     private val appCoroutineScope: CoroutineScope,
     private val activeCallManager: ActiveCallManager,
+    private val languageTagProvider: LanguageTagProvider,
```

Comment:
Introduce a LanguageTagProvider because unit test does not like using `LocalConfiguration`